frontend/src/components/Dashboard/Opening/RecentOpeningsDataTable/index.tsx

```diff
@@ -0,0 +1,331 @@
+                    header.selected ? (
+                      <TableCell
+                        key={header.key}
+                        className={
+                          header.key === "actions" && showSpatial ? "p-0" : null
+                        }
+                      >
+                        {header.key === "statusDescription" ? (
+                          <StatusTag code={row[header.key]} />
+                        ) : header.key === "actions" ? (
+                          <>
+                            <>
+                            <Button
+                              hasIconOnly
+                              iconDescription="View"
+                              tooltipPosition="auto"
+                              kind="ghost"
+                              onClick={() => console.log(row.openingid)}
+                              renderIcon={Icons.View}
+                              size="md"
+                            />
+                            <Button
+                              hasIconOnly
+                              iconDescription="Document Download"
+                              tooltipPosition="auto"
+                              kind="ghost"
+                              onClick={() => null}
+                              renderIcon={Icons.DocumentDownload}
+                              size="md"
+                            />
+                          </>
+                            <CheckboxGroup
+                              orientation="horizontal"
+                              className="align-items-center justify-content-start"
+                            >
+                              {/* Checkbox for selecting rows */}
+                              {showSpatial && (
+                                <Tooltip
+                                  className="checkbox-tip"
+                                  label="Click to view this opening's map activity."
+                                  align="bottom-left"
+                                  autoAlign
+                                >
+                                  <div className="mb-2 mx-2">
+                                    <Checkbox
+                                      id={`checkbox-label-${row.openingId}`}
+                                      checked={selectedRows.includes(
+                                        row.openingId
+                                      )}
+                                      onChange={() =>
+                                        handleRowSelectionChanged(row.openingId)
+                                      }
+                                    />
+                                  </div>
+                                </Tooltip>
+                              )}
+                            </CheckboxGroup>
+                          </>
+                        ) : header.header === "Category" ? (
+                          row["categoryCode"] +
```

Comment:
Switch to the TruncatedText component as used in [SearchScreenDataTable](https://github.com/bcgov/nr-silva/blob/main/frontend/src/components/SilvicultureSearch/Openings/SearchScreenDataTable/index.tsx#L376) to keep the same behaviour as in the other screen

Reply:
done